### ISSUE TITLE
False positive on 'Rails/SkipsModelValidations' cop for 'FileUtils.touch'

### ISSUE THREAD

**Issue by bilal-khan** [bbatsov/rubocop#4260]
## Expected behavior

Not to flag this as an error

## Actual behavior

```
Avoid using touch because it skips validations.
    FileUtils.touch logfile
```

## Steps to reproduce the problem

Use the above FileUtils command

## RuboCop version

```
$ rubocop -V
0.48.1 (using Parser 2.4.0.0, running on ruby 2.4.1 x86_64-darwin16)
```

**Comment by bbatsov** [comment]
Ops, that's one silly mistake we made.

**Comment by bilal-khan** [comment]
😄 easy mistake to make!

**Comment by bbatsov** [comment]
True, seems this cop treats a bunch of methods in more or less the same manner. It will require a bit of work to split those to class methods and instance methods.
